helpers/helper-translation-discussion.php

```diff
@@ -29,7 +29,15 @@ class Helper_Translation_Discussion extends GP_Translation_Helper {
 	 * @since 0.0.1
 	 * @var bool
 	 */
-	public $has_async_content = false;
+	public $has_async_content = true;
+
+	/**
+	 * Indicates whether the helper should be loaded inline.
+	 *
+	 * @since 0.0.1
```

Comment:
We are marking the new properties as 0.0.2. The 0.0.1 properties come from DotCom.